django_declarative_apis/machinery/filtering.py

```diff
@@ -9,6 +9,11 @@
 import inspect
 import types
 
+try:
+    from django.core.exceptions import FieldDoesNotExist
```

Comment:
We can drop this too if we don't care about Django 2.2 compatibility:

https://docs.djangoproject.com/en/4.0/releases/3.1/#miscellaneous-1